### PR TITLE
{Legal} Add paramiko to notice file

### PR DESCRIPTION
**Pull request by fengzhou-msft**
**Description**

The `paramiko` dependency package uses LGPL-2.1 license and needs to be added in the `NOTICE.txt` file.

The content is generated by ADO Component Governance.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Legal

**Review by jiasli — APPROVED** [review]

File: NOTICE.txt [hunk 1]
```diff
@@ -1932,6 +1932,220 @@ That's all there is to it!
 
 ---------------------------------------------------------
 
+paramiko 2.6.0 - LGPL-2.1-only
```
Comment (yungezz) [inline NOTICE.txt:1935]: this will be a manual maintain effort in future right? Is there way to track the route work, such as in doc etc.

**Review by yungezz — APPROVED** [review]

**Comment by qwordy** [comment]
Does it conflict with the current MIT license of Azure CLI?

File: NOTICE.txt (hunk 1 above)
Comment (fengzhou-msft) [reply NOTICE.txt:1935]: The content can be generated on ADO. I will add a OneNote about how to do it.

**Comment by fengzhou-msft** [comment]
> Does it conflict with the current MIT license of Azure CLI?

LGPL-2.1 is more strict when you need to modify its source code. We're using the package as it is, so it won't affect Azure CLI's distribution license in general.

**Pull request merged**